app/views/home/_constituency_form.html.haml

```diff
@@ -0,0 +1,14 @@
+= form_with url: pre_login_path, id: "login-form", local: true do |f|
+
+
+  %p.choose-constituency
+    My constituency is
+    = f.collection_select :constituency_ons_id, constituencies,
+                          :ons_id, :name,
+                          selected: default_constituency_ons_id,
+                          prompt: true
+
+  = render partial: "users/postcode_field"
```

Comment:
You might need to put this in a div to make sure it is rendered with the right padding.